sktime/clustering/kmeans.py

```diff
@@ -37,13 +40,13 @@ def __init__(self, n_clusters, cluster_centers, metric, tolerance, inertia=np.in
             The metric that was used
         tolerance : float
             Tolerance which was used as convergence criterium.
-        inertia : float, optional, default=np.inf
-            Value of the cost function at the end of the estimation.
+        inertias : (t,) ndarray, optional, default=None
```

Comment:
I don't really understand from the docs what happens if I put in an array instead of `None`

Reply:
added clarifying docstring, this is basically just data that falls out of the estimator. The aim is to have model classes which can be instantiated without estimator as well - in this case based on cluster centers and metric.

Reply:
cool!